src/js/components/tasks/createForm.tsx

```diff
@@ -215,6 +223,14 @@ const CreateTaskModal = ({
       ]}
     >
       <form onSubmit={doSubmit} className="slds-form slds-p-around_large">
+        {epic ? (
+          ''
+        ) : (
+          <p className="slds-align_absolute-center">
+            You are creating one task for this project. To group multiple tasks
+            together, create an Epic.
+          </p>
+        )}
```

Comment:
This needs translation, and we can clean up the logic just a bit:

```suggestion
        {!epic && (
          <p className="slds-align_absolute-center slds-m-bottom_small">
            {i18n.t(
              'You are creating one Task for this Project. To group multiple Tasks together, create an Epic.',
            )}
          </p>
        )}
```

Reply:
@oluoluoxenfree I think this hasn't been addressed yet?

Reply:
missed this comment, adding this in now.